modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,69 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
```

Comment:
When does a user use this method?
The second bullet point says: 'If you do not use the `vfs` storage driver,...' - would it be correct to start with this method saying: 'If you use the `vfs` storage driver,...'

Reply:
I don't think it would be appropriate to start the first case with "If you use the `vfs` storage driver...". That's because the first point is the most generic case, which covers almost everything except the scenario of using `vfs` storage drivers. In other words, the points are not about mutually exclusive scenarios, where either you use the `vfs` storage driver or you don't use it.